man/systemd.exec.xml

```diff
@@ -365,6 +365,22 @@
         <xi:include href="version-info.xml" xpointer="v233"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BindJournalSockets=</varname></term>
+
+        <listitem><para>Takes a boolean argument. If true, sockets from <citerefentry>
+        <refentrytitle>systemd-journald.socket</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+        will be bind mounted into the mount namespace. This is particularly useful when a different rootfs
+        is used (through <varname>RootDirectory=</varname> or <varname>RootImage=</varname>), as
+        <varname>MountAPIVFS=yes</varname> mounts an empty <filename>/run/</filename> inside the new root.</para>
+
+        <para>This option is implied when <varname>LogNamespace=</varname> is used,
+        when <varname>MountAPIVFS=yes</varname>, or when <varname>PrivateDevices=yes</varname> is used
+        in conjunction with either <varname>RootDirectory=</varname> or <varname>RootImage=</varname>.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
```

Comment:
I don't grok the usecase for an explicit switch?

Reply:
Hmm, so in the majority of cases this would be implied, but it feels very weird if this is directly added to `MountAPIVFS=yes`. E.g. people might use `TemporaryFileSystem=/`, and they might only want the journal sockets but no other API file systems. Also, if they want something like `StandardOutput/Error=null`, they'll probably also close down journal socket access.

Reply:
but if they do TemporaryFileSystem=/ then they are going down a more manual route anyway, so they could also manually bind the sockets? why give them a high-level knob if they go down the low-level route apparently?